feature-utils/poly-sparql/src/triple-generation.js

```diff
@@ -0,0 +1,66 @@
+import { v4 as uuid } from "uuid";
+
+export function jsObjectToTriplesString(
+    subject,
+    predicate,
+    jsObj,
+    blankNodeString = "_:" + uuid()
+) {
+    let triplesString = `${subject} ${predicate} ${blankNodeString} `;
+    for (let [key, value] of Object.entries(jsObj)) {
+        const keyPredicate = "poly:" + key;
+        if (typeof value === "object") {
```

Comment:
Maybe use an `and` expression instead of two ifs?

Reply:
But I specifically want to tell the user that arrays of arrays are not supported